server/src/main/java/com/objectcomputing/checkins/services/kudos/Kudos.java

```diff
@@ -0,0 +1,188 @@
+package com.objectcomputing.checkins.services.kudos;
+
+import io.micronaut.core.annotation.Introspected;
+import io.micronaut.core.annotation.Nullable;
+import io.micronaut.data.annotation.AutoPopulated;
+import io.micronaut.data.annotation.DateCreated;
+import io.micronaut.data.annotation.TypeDef;
+import io.micronaut.data.annotation.sql.ColumnTransformer;
+import io.micronaut.data.model.DataType;
+import io.swagger.v3.oas.annotations.media.Schema;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
+
+import java.time.LocalDate;
+import java.util.Objects;
+import java.util.UUID;
+
+@Entity
+@Introspected
+@Table(name = "kudos")
+public class Kudos {
+
+    @Id
+    @Column(name = "id")
+    @AutoPopulated
+    @TypeDef(type = DataType.STRING)
+    @Schema(description = "the id of the kudos", required = true)
```

Comment:
The `required` attribute is deprecated and can be removed from all of these `@Schema` annotations (it's automatically calculated now, with `requiredMode` used when overriding the value is required)